### ISSUE TITLE
Demo is not working in IE?

### ISSUE THREAD

**Issue by cc-austin** [tim-peterson/videojs-playlist#21]
Hi Tim,

Thank you for making a great video.js playlist!  I would love to use your plugin on one of the websites I'm working on.  The demo works really well in Chrome and Firefox, but there seems to be an issue in IE?  I tested your demo on my IE 11, and it has this message of "The video could not be loaded, either because the server or network failed or because the format is not supported".  Do you know if this is an issue relating to the file format or the plugin itself?  Thanks!

**Comment by tim-peterson** [comment]
It is probably the file types. I don't have easy access to IE, so I'm sorry that's its not working there. To test, perhaps you can swap in non-copyrighted videos from the Video.js website or elsewhere that are known to work on IE?

**Comment by cc-austin** [comment]
Hi Tim, thank you for your response!  You are right, I swapped in other videos elsewhere and the playlist works in IE!

FYI - I later bumped into another problem.  After I swapped those videos again to my own videos in various formats, the same error messages appeared in IE and I couldn't figure out why.  I was testing locally on my computer the entire time and decided to upload all the files onto the web server and it somehow solves this second problem!

Thank you again Tim!
